crates/fluvio-cli/src/client/hub/list.rs

```diff
@@ -5,13 +5,14 @@ use clap::Parser;
 
 use fluvio_extension_common::Terminal;
 use fluvio_hub_util as hubutil;
-use hubutil::PackageList;
+use hubutil::PackageListMeta;
 use hubutil::http;
 
 use crate::{CliError, Result};
+use crate::client::hub::get_hub_access;
 use crate::common::OutputFormat;
 
-const API_LIST: &str = "hub/v0/list";
+const API_LIST_META: &str = "hub/v0/list_with_meta";
```

Comment:
is this compatible with old CLI?

Reply:
Yes. The 0.10.1 and older fluvio hub list uses hub/v0/list - so it will continue to operate until a user upgrades the cli 0.10.2, then the cli will access list_with_meta.
list_with_meta was already being used by the ui and the changes are backward compatible with the current ui (which works but doesn't display private / public field). The updated ui will parse and display the visiblity field as private or public.